packages/wrangler/src/index.tsx

```diff
@@ -1477,9 +1436,11 @@ export async function main(argv: string[]): Promise<void> {
           },
           async ({ key, ttl, expiration, ...args }) => {
             const namespaceId = getNamespaceId(args);
+            // One of `args.path` and `args.value` must be defined
```

Comment:
So should not possible to be an `undefined` state for both `args.path` and `args.value`? 

just clarifying.

Reply:
Yes due to:

```ts
.check(demandOneOfOption("value", "path"));
```